libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.ts

```diff
@@ -50,6 +50,11 @@ export class SkyDataManagerColumnPickerComponent implements OnDestroy, OnInit {
         (col) => col.isSelected
       );
     }
+
+    this.isAnyDisplayedColumnSelected =
```

Comment:
This exact logic is duplicated multiple times. It should me moved into a private function.

Reply:
I resolve this issue in [my next PR](https://github.com/blackbaud/skyux/pull/349). I'll send and assign that PR to you after this one is merged.